cypress/integration/SequencePage.spec.js

```diff
@@ -295,4 +295,68 @@ context('SequencePage', () => {
     cy.get('section').first().contains('Bauchbinde IN:00:00:15:00')
     cy.get('section').first().contains('Bauchbinde OUT:00:00:23:00')
   })
+
+  it('tests if the delete data button is deactivated at start', () => {
+    cy.get('[data-cy=deleteStorageButton]').should(
```

Comment:
You are testing if the button is disabled by checking the fill. Can't you use the "disabled" attribute on the button?

Reply:
not in this case, because it's a svg that is clickable and not a button.